.github/workflows/linting-backend.yml

```
@@ -0,0 +1,27 @@
+# https://deno.land/manual@v1.19.2/continuous_integration
+
+name: CI - Linting - Backend
+
+on:
+  pull_request:
+    paths:
+      - 'backend/**'
+  workflow_dispatch:
+
+jobs:
+  lint-backend:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+
+      - name: Format the backend
+        run: cd backend/ && deno fmt --config deno.json
+
+      - name: Lint the backend
+        run: cd backend/ && deno lint --config deno.json
+
+      - name: Test the backend
+        run: cd backend/ && deno test --config deno.json --import-map=../import_map.json
```

I added the `--check` option to `deno fmt` to fail on improper formatted files. Without this it will just format (change) files and exit with code 0. Second suggestion is to use the `working-directory` key instead of `cd` 😊 

```suggestion
      - name: Check backend formatting
        working-directory: backend
        run: deno fmt --config deno.json --check

      - name: Lint the backend
        working-directory: backend
        run: deno lint --config deno.json

      - name: Test the backend
        working-directory: backend
        run: deno test --config deno.json --import-map=../import_map.json
```

---

Yes, you are right. The `--check` flag mysteriously disappeared when I switched between `deno` and `trex`.